data/json/recipes/recipe_food.json

```diff
@@ -9352,5 +9352,51 @@
       [ "thyme", 1 ]
     ]
   ]
+},{
+  "type" : "recipe",
+  "result": "seed_canola",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_OTHER",
+  "skill_used": "survival",
+  "difficulty": 2,
+  "time": 5000,
+  "reversible": false,
+  "autolearn": true,
+  "qualities" : [
+    {"id":"CUT","level":1,"amount":1}
+  ],
+  "components": [
+    [
+      [ "canola", 1 ]
+    ]
+  ]
+},{
```

Comment:
We already have a nearly-identical cooking oil recipe at L:203. Why not just give it ````seed_canola```` as a possible component?